ompi/mca/mtl/psm2/mtl_psm2.c

```diff
@@ -171,6 +174,15 @@ int ompi_mtl_psm2_module_init(int local_rank, int num_local_procs) {
     /* register the psm2 progress function */
     opal_progress_register(ompi_mtl_psm2_progress);
 
+#if OPAL_CUDA_SUPPORT
+    ompi_mtl_psm2.super.mtl_flags |= MCA_MTL_BASE_FLAG_CUDA_INIT_DISABLE;
+
+    cuda_env = getenv("PSM2_CUDA");
+    if (!cuda_env || ( strcmp(cuda_env, "0") == 0) )
+        opal_output(0, "Warning: If running with device buffers, there is a"
+                    " chance the application might fail. Try setting PSM2_CUDA=1.\n");
```

Comment:
Please use `opal_show_help()` for these kinds of messages (because `opal_show_help()` deduplicates such messages when outputting to `mpirun`).  There's also very little context given in this message to inform the user that the message is being emitted from the Open MPI PSM2 code.

Reply:
ok, will fix this